check/configuration/supported/ansible-lint.md

```diff
@@ -0,0 +1,28 @@
+---
+description: Ansible-Lint
+---
+
+# Ansible-lint
+
+Ansible-lint is a linter for infra.
```

Comment:
We should be more specific here imo (e.g. ansible-lint is an infra linter for ansible playbooks and rules)